tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -725,7 +725,7 @@ public function theAdminHasRetrievedMembersListOfGroupUsingTheGraphApi(string $g
 	}
 
 	/**
-	 * @When /^the user "([^"]*)" creates a new user using GraphAPI with the following settings:$/
+	 * @When /^the user "([^"]*)" creates a new user with the following attributes using GraphAPI:$/
```

Comment:
to maintain consistency
```suggestion
	 * @When /^the user "([^"]*)" creates a new user with the following attributes using the Graph API:$/
```